shell/assets/translations/en-us.yaml

```diff
@@ -4991,6 +4995,10 @@ promptRollback:
   multipleWorkloadError: "Only one workload can be rolled back at a time."
   singleRevisionBanner: There are no revisions to roll back to.
 
+promptRedeploy:
+  title: Are you sure?
```

Comment:
I think that the title should provide more context about the action. Something like "Redeploy workload?" or "Redeploy workload {workloadName}" better communicates the impact than "Are you sure?"